component/admin/sql/install/mysql/install.sql

```diff
@@ -1,5 +1,45 @@
 SET FOREIGN_KEY_CHECKS = 0;
 
+-- -----------------------------------------------------
+-- Table `#__redcore_country`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `#__redcore_country` (
+  `id` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `alpha2` CHAR(2) NOT NULL,
+  `alpha3` CHAR(3) NOT NULL,
+  `numeric` SMALLINT(3) UNSIGNED NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `idx_name` (`name` ASC),
+  UNIQUE KEY `idx_alpha2` (`alpha2` ASC),
+  UNIQUE KEY `idx_alpha3` (`alpha3` ASC),
+  UNIQUE KEY `idx_numeric` (`numeric` ASC)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- -----------------------------------------------------
+-- Table `#__redcore_currency`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `#__redcore_currency` (
+  `id` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `alpha3` CHAR(3) NOT NULL,
+  `numeric` SMALLINT(3) UNSIGNED NOT NULL,
+  `symbol` VARCHAR(255) NOT NULL,
+  `symbol_position` TINYINT(1) NOT NULL DEFAULT '1' COMMENT 'display currency symbol before (0) or after (1) price',
+  `decimals` TINYINT(1) NOT NULL DEFAULT '0' COMMENT 'number of decimals to show in prices',
+  `state` TINYINT(1) NOT NULL DEFAULT '1' COMMENT 'disabled(0) / enabled(1)',
+  `blank_space` TINYINT(1) NOT NULL DEFAULT '1' COMMENT 'display a blank space between the currency symbol and the price',
+  `decimal_separator` VARCHAR(1) NOT NULL DEFAULT ',',
```

Comment:
We had some discussion in b2b a few months back, and both decimal_separator and thousands_separator should be in country table, not in currency - even a language config would make more sense